ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -512,14 +520,16 @@ public static BlobProperties buildBlobProperties(Map<String, Object> args) throw
    * @throws RestServiceException if the {@link GetBlobOptions} could not be constructed.
    */
   public static GetBlobOptions buildGetBlobOptions(Map<String, Object> args, SubResource subResource,
-      GetOption getOption) throws RestServiceException {
+      GetOption getOption, long blobSegment) throws RestServiceException {
```

Comment:
add java doc for `blobSegment` (or maybe call it `blobSegmentIndex`?)

Reply:
done